src/lang/lang_values.ml

```diff
@@ -1059,8 +1061,8 @@ and apply f l =
   (* Record error positions. *)
   let f pe =
     try f pe with
-      | Runtime_error (poss, e) ->
-          raise (Runtime_error (Option.to_list pos @ poss, e))
+      | Runtime_error err ->
+          raise (Runtime_error { err with pos = Option.to_list pos })
```

Comment:
No, no, no, the idea was that we keep the stacktrace! Revert this!

Reply:
Hmm. I must be missing something here. All I'm intending to do is structure the arguments, more than 3 I like to use records. I think you're referring to dropping the `@ poss`, I'll add it back.

Reply:
Yes, the `@`!